src/lib/hooks/use-best-quote/index.ts

```diff
@@ -40,7 +39,7 @@ export const useBestQuote = (
   outputToken: Token,
 ) => {
   const publicClient = usePublicClient()
-  const { provider, signer } = useWallet()
+  const { address, jsonRpcProvider, provider } = useWallet()
```

Comment:
Only place json rpc provider is now still used but this we will need because of passing to the FlashMint SDK